hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -2370,4 +2387,22 @@ void sortDatanodes(String clientMachine, OmKeyInfo... keyInfos) {
     }
     return nodeSet;
   }
+
+  private void slimLocationVersion(OmKeyInfo... keyInfos) {
+    if (keyInfos != null) {
+      for (OmKeyInfo keyInfo : keyInfos) {
+        OmKeyLocationInfoGroup key = keyInfo.getLatestVersionLocations();
+        if (key == null) {
+          LOG.warn("No location version for key {}", keyInfo);
+          continue;
+        }
+        int keyLocationVersionLength = keyInfo.getKeyLocationVersions().size();
+        if (keyLocationVersionLength <= 1) {
+          continue;
+        }
+        keyInfo.setKeyLocationVersions(keyInfo.getKeyLocationVersions()
```

Comment:
Thanks for the review. 
I think it's caused by the definition of `keyLocationVersions: List<OmKeyLocationInfoGroup>`, the `slimLocationVersion` here is just trying to get the last value of `keyLocationVersions`, which is an `OmKeyLocationInfoGroup`. As for the problem that all versions are stored in `OmKeyLocationInfoGroup.locationVersionMap`, I think we can raise another ticket to check the design?

For subList, since `setKeyLocationVersions` accepts a List, seems we can't return the last value directly.

Reply:
>I think we can raise another ticket to check the design?

Makes sense to me. Let us raise a new Jira and discuss this.

>For subList, since setKeyLocationVersions accepts a List, seems we can't return the last value directly.
Got it.